llvm/lib/Support/CommandLine.cpp

```diff
@@ -2353,9 +2353,31 @@ sortSubCommands(const SmallPtrSetImpl<SubCommand *> &SubMap,
 
 namespace {
 
+#ifdef SYCLomatic_CUSTOMIZATION
+/// HelpCategory defines various category groups for dpct options
+enum class HelpCategory {
+  HC_All,
+  HC_Basic,
+  HC_Advanced,
+  HC_CodeGen,
+  HC_ReportGen,
+  HC_BuildScript,
+  HC_QueryAPI,
+  HC_Warnings,
+  HC_HelpInfo,
+  HC_InterceptBuild,
+  HC_Examples
+};
+#endif // SYCLomatic_CUSTOMIZATION
+
+// clang-format off
```

Comment:
L2374 should be protected by SYCLomatic_CUSTOMIZATION. 
L2373 "clang-format off " should be removed.